src/processor/LakeOfFHIR.ts

```diff
@@ -16,7 +16,9 @@ export class LakeOfFHIR implements utils.Fishable {
    * @returns {WildFHIR[]}
    */
   getAllStructureDefinitions(): WildFHIR[] {
-    return this.docs.filter(d => d.content.resourceType === 'StructureDefinition');
+    return this.docs
+      .filter(d => d.content.resourceType === 'StructureDefinition')
+      .filter(d => !this.isSpecialization(d)); // TODO: Remove this filter when full support for LogicalModels and Resources is implemented
```

Comment:
Somehow, I didn't realize until you pointed it out here: Profiles and Extensions are constraints, while Logicals and Resources are specializations. A delightfully concise implementation.

Reply:
Data types are also `specialization`s (just FYI).

Reply:
Except a few variants on `Quantity` that are actually profiles...